dsa-23au/java-dsa/arrays-links/src/test/java/dev/codewithfriends/ArrayWrapperTest.java

```diff
@@ -1,14 +1,39 @@
 package dev.codewithfriends;
 
-import static org.junit.Assert.assertTrue;
 import static org.junit.Assert.assertEquals;
-
 import org.junit.Test;
 import java.util.List;
+import java.util.ArrayList;
 
 /**
- * Unit test for simple App.
+ * Unit test for ArrayWrapper class.
  */
+public class ArrayWrapperTest {
```

Comment:
It is easy to follow and understand what the JUnit is doing. He creates a String array and then creates an ArrayList. The ArrayList loops through and uses the add method that he made. The JUnit then compares the string array and the Arraylist that used his add method.